docs/references/react/use-user.mdx

```diff
@@ -1,49 +1,47 @@
 ---
 title: '`useUser()`'
-description: The useUser() hook is used to get the current user object and to update the user's data on the client-side.
+description: Access and manage the current user's data in your React application with Clerk's useUser() hook.
 ---
 
-The `useUser()` hook is a convenient way to access the current user's [`User`](/docs/references/javascript/user/user) object, which holds all of the information for a single user of your application and provides a set of methods to manage their account. This hook also provides boolean values to check if the user is signed in and if Clerk has loaded and initialized.
+The `useUser()` hook provides access to the current user's [`User`](/docs/references/javascript/user/user) object, which contains all the data for a single user in your application and provides methods to manage their account. This hook also allows you to check if the user is signed in and if Clerk has loaded and initialized.
 
-## `useUser()` returns
+## Returns
 
 <Properties>
-  - `isSignedIn`
+  - `isLoaded`
   - `boolean`
 
-  A boolean that returns `true` if the user is signed in.
+    A boolean that indicates whether Clerk has completed initialization. Initially `false`, becomes `true` once Clerk loads.
 
   ---
 
-  - `isLoaded`
+  - `isSignedIn`
   - `boolean`
 
-  A boolean that until Clerk loads and initializes, will be set to `false`. Once Clerk loads, `isLoaded` will be set to `true`.
+  A boolean that returns `true` if the user is signed in.
 
   ---
 
   - `user`
   - [`User`](/docs/references/javascript/user/user)
 
-  The `User` object for the currently active user. If the user is not signed in, `user` will be `null`.
+  The `User` object for the current user. If the user isn't signed in, `user` will be `null`.
 </Properties>
 
 ## How to use the `useUser()` hook
 
-### Retrieve the current user data with the `useUser()` hook
-
-The following example demonstrates how to use the `useUser()` hook to access the `user` object, which includes the current user's data, like the user's full name. The `isLoaded` and `isSignedIn` properties are used to handle the loading state and to check if the user is signed in, respectively.
+### Get the current user
 
-For more information on the `User` object, see the [`reference`](/docs/references/javascript/user/user).
+Use `useUser()` to access the [`User`](/docs/references/javascript/user/user) object, which contains the current user's data such as their full name. The `isLoaded` and `isSignedIn` values are used to handle the loading state and to check if the user is signed in, respectively.
```

Comment:
actually, I don't see anything wrong with the copy before, and its a common syntax structure we use across the docs: "the following example demonstrates..."
if you feel this is cleaner, that's fine, but isLoaded and isSignedIn aren't values, they're properties that are set to boolean values.